src/main/java/core/basesyntax/service/WriterReportToCsvImpl.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax.service;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Map;
+
+public class WriterReportToCsvImpl implements WriterReportToCsv {
+    private static final String TITLE = "fruit,quantity";
+
+    @Override
+    public void writeReport(Map<String, Integer> storage, String outputFilePath) {
```

Comment:
```suggestion
    public void writeReport(String outputFilePath) {
```
remove this parameter you can call it directly